cloudinit.tpl

```diff
@@ -0,0 +1,16 @@
+#cloud-config
+yum_repos:
+  docker-ce:
+    name: docker-ce
+    description: Docker CE Repository
+    baseurl: "https://download.docker.com/linux/centos/7/$basearch/stable"
+    enabled: yes
+    gpgcheck: yes
+    gpgkey: https://download.docker.com/linux/centos/gpg
+packages:
+- docker-ce
+runcmd:
+  - [ systemctl, daemon-reload ]
+  - [ systemctl, enable, docker.service ]
+  - [ systemctl, start, --no-block, docker.service ]
+  - setenforce 0
```

Comment:
Do we expect only to support RHEL/Centos for this feature?

Reply:
till now yes as these are the only supported OSes

Reply:
We are getting Ubuntu support in RI-13: https://jira.mesosphere.com/browse/DCOS-49340

Since we're expecting this, should we add the boilerplate in place so we avoid a breaking change and put this template in its own module that will support other operating systems? Ansible and Ubuntu is ❤️

Reply:
Thanks for sharing. Sure that makes sense.